packages/perseus-editor/src/item-editor.tsx

```diff
@@ -74,16 +76,13 @@ class ItemEditor extends React.Component<Props> {
     };
 
     serialize: (options?: any) => {
-        answerArea: any;
-        itemDataVersion: {
-            major: number;
-            minor: number;
-        };
-        question: any;
+        answerArea: PerseusAnswerArea;
+        itemDataVersion: Version;
+        question: PerseusRenderer;
     } = (options: any) => {
         return {
-            question: this.questionEditor.current?.serialize(options),
-            answerArea: this.itemExtrasEditor.current?.serialize(),
+            question: this.questionEditor.current!.serialize(options),
+            answerArea: this.itemExtrasEditor.current!.serialize(),
```

Comment:
**Question**: Is there really never a case when these could be null? (Re: the `!.`)